ant/__init__.py

```diff
@@ -20,4 +20,6 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+from __future__ import absolute_import, print_function
```

Comment:
Yes, it was related to this. Maybe you could merge now if the rest is OK, and i'll try to add `unicode_literals` later ? I've also fixed issues in unit tests for this branch.